settings.gradle.kts

```diff
@@ -1,5 +1,7 @@
 rootProject.name = "snakeyaml-engine-kmp"
 
+include("benchmark")
```

Comment:
I'm a big fan of using a common prefix in subprojects, so that they are visually grouped together in the file list. What about renaming the benchmarks to `snake-kmp-benchmarks`?

Reply:
I have no preferences here) Will rename the project